# Request
sensor_msgs/Image image
string text_prompt
float32 box_threshold
float32 text_threshold

---

# Response
CompressedMask[] masks